src/pug/blog/typing_methods.pug

```diff
@@ -0,0 +1,338 @@
+      td
+        div #[b Primary:] L1 (Starts character recognition, try to match character on release).
+        div &nbsp;
+        div #[b Secondary:] Gyro (Moves the virtual "pen").
+        div &nbsp;
+        div: i (No grouping)
+      td
+        ul
+          li No preview.
+          li As in writing on the air.
+          li Firmware implementation would be very hard to develop.
+          li Would require to adapt to user's specific way to write characters?.
+
+  hr
+  h3 5. General solutions (further evaluation)
+  p From all the methods evaluated, we decided to do further testing with 2 of them: #[em Daisywheel] and #[em Glyph-stick], as we found these the most promising solutions, being the easier to understand and learn, not requiring preview-and-commit, and being easier to remember.
+
+  h4 5.1. Daisywheel
+  ul
+    li Based on Steam Controller / Steam Input daisywheel method.
+    li Will be available on the #[+a('/alpakka/manual/profile_desktop') Desktop profile].
+    li The #[em thumbstick] direction pre-selects a group of 4 characters, final selection is made with #[em ABXY] buttons.
+    li Eventually, it should be possible to edit the grouping in the profiles.
+    li Classic variant:
+    ul
+      li Alphabetical grouping, same than the original daisywheel.
+      li Order of face buttons is #[em X], #[em Y], #[em B], #[em A].
+    li Tweaked variant:
+    ul
+      li Keypad-style grouping, but adapted for having no center key.
+      li Order of face buttons is #[em A], #[em B], #[em X], #[em Y].
+      li #[em O] and #[em U] are relocated so all vowels are on the #[em A button].
+      li #[em X] and #[em Y] are relocated on the #[em X button] and #[em Y button].
+
+  +figure(
+    null,
+    'Daisywheel classic (Steam Controller)',
+    '/static/blog/daisywheel_classic.png',
+    'big inline'
+  )
+  +figure(
+    null,
+    'Daisywheel tweaked (Input Labs)',
+    '/static/blog/daisywheel_tweaked.png',
+    'big inline'
+  )
+
+  h4 5.2. Glyph-stick
+  ul
+    li #[em Thumbstick] is used for basic pattern recognition (glyphs).
+    li It is usable with a single finger, or even no fingers.
+    li Glyph patterns are designed to resemble the characters they represent, so they are easier to remember.
+    li All glyphs start/end from the center into a cardinal direction (left, right, up, down).
+    li Glyph is evaluated when returning to center.
+    li Shorter patterns are used for more frequent characters (in english).
+    li Eventually, it should be possible to edit glyphs in the profiles.
+  +figure(
+    null,
+    '',
+    '/static/blog/glyph-stick.png',
+    'big'
+  )
+
+  hr
+  h3 6. Next steps
+  p The plan is to release these new input methods as part of the Alpakka firmware, possibly within a #[+ax(org.fw_releases) Beta release] #[i (too be announced soon)], with the goal of getting feedback from testers, iterate over the implementations, or even discard the methods completely.
```

Comment:
Rephrase to something like "The plan is to gather feedback, and to do alpha (internal) and beta testing with our testers. Based on the results, some of these mechanisms may become part of future firmware releases. These will then be iterated and improved accordingly."